app/controllers/api/v1/statuses_controller.rb

```diff
@@ -48,11 +48,18 @@ def create
       scheduled_at: status_params[:scheduled_at],
       application: doorkeeper_token.application,
       poll: status_params[:poll],
+      safeguard_mentions: status_params[:safeguard_mentions],
```

Comment:
I believe `protected_mentions` would be a much clearer and cleaner name for this feature.

Reply:
Maybe `allowed_mentions` would be even better? Those are the accounts you expect to see mentioned.